versions.gradle

```diff
@@ -1,27 +1,24 @@
 ext {
-
     // Project
     libVersion = "3.0.0"
     compileSdkVersion = 30
-    targetSdkVersion = 30
+    buildToolsVersion = "30.0.3"
```

Comment:
Why we are defining a buildToolsVersion when we can let AS do this? what are the pros/cons?

Reply:
Studio uses an older version by default, so we can specify using a newer version of the build tools.

Reply:
Got it, why we want the newer version? 

There is a bad part of using the older version? because is one more dependency to maintain